<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Players</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Player:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="playerListBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="viewPlayerButton">
       <property name="text">
        <string>Camera to Player's View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="movePlayerButton">
       <property name="text">
        <string>Move Player to Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="playerTab">
      <attribute name="title">
       <string>Player</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="inventoryGroupBox">
         <property name="title">
          <string>Inventory</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="allPropertiesTab">
      <attribute name="title">
       <string>All Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="PropertyListWidget" name="playerPropertiesWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nbtTreeTab">
      <attribute name="title">
       <string>NBT Tree</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="NBTEditorWidget" name="nbtEditor" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NBTEditorWidget</class>
   <extends>QWidget</extends>
   <header>nbteditorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PropertyListWidget</class>
   <extends>QTreeWidget</extends>
   <header>propertylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
